src/components/SignUp/SignUp.js

```diff
@@ -37,14 +36,6 @@ class SignUp extends Component {
     })
   }
 
-  validateEmail = (email) => {
-    // one @ symbol,
-    //  at least one . after @ symbol
-    //  . cannot be next to another .
-    // . cannot be at beginning or end of string
-    // let regex = \@
-  }
-
```

Comment:
I think this part at least would be doable without regex and just string comparison?